packages/widgets/src/components/ExchangeAccount/ExchangeAccount.spec.tsx

```diff
@@ -0,0 +1,136 @@
+import { MemoryRouter, Route, Routes } from 'react-router-dom';
+import userEvent from '@testing-library/user-event';
+import { ExchangeAccount } from './ExchangeAccount';
+import { dataValues } from './helpers';
+
+const commonProps = {
+  platform: dataValues[0].platform,
+  title: dataValues[0].title,
+  data: dataValues[0].data,
+};
+
+const ResponsivePieMock = () => <div data-testid="responsive-pie"></div>;
+
+jest.mock('@nivo/pie', () => ({
+  PieCustomLayerProps: {},
+  ResponsivePie: ResponsivePieMock,
+}));
+
+describe('ExchangeAccount', () => {
+  it('Renders with platform', () => {
+    const { getByText } = render(
+      <ExchangeAccount
+        {...commonProps}
+        deleteOnClick={jest.fn()}
```

Comment:
Could we move `deleteOnClick={jest.fn()}` to the commonProps also? And rewrite this prop when we need it

Reply:
Yes, you are right. Done.